frontend/app/components/context-menus/context-menu.service.ts

```diff
@@ -54,21 +57,21 @@ export class ContextMenuService {
 
   // Return the active context menu, if any
   public get active():ContextMenu|null {
-    return this._active_menu;
+    return this.active_menu;
   }
 
-  public close(disableFocus:boolean = false) {
+  public close(disableFocus:boolean = false):Promise<void> {
     if (!this.active) {
-      return this.$q.when(true);
+      return this.$q.when(void 0);
```

Comment:
Used be a `true`, now `void 0`. Are you sure? Also, I find `void 0` uncommon. Why not `null` or `undefined`?